src/t_string.c

```diff
@@ -563,8 +571,10 @@ void msetGenericCommand(client *c, int nx) {
 
     int setkey_flags = nx ? SETKEY_DOESNT_EXIST : 0;
     for (j = 1; j < c->argc; j += 2) {
-        c->argv[j+1] = tryObjectEncoding(c->argv[j+1]);
-        setKey(c, c->db, c->argv[j], c->argv[j + 1], setkey_flags);
+        robj *val = tryObjectEncoding(c->argv[j+1]);
+        setKey(c, c->db, c->argv[j], &val, setkey_flags);
+        incrRefCount(val);  /* adding ... */
+        c->argv[j+1] = val;
```

Comment:
```suggestion
        c->argv[j+1] = tryObjectEncoding(c->argv[j+1]);
        setKey(c, c->db, c->argv[j], &c->argv[j+1] , setkey_flags);
        incrRefCount(c->argv[j+1]);  /* adding ... */
```
unfinished comment `adding`?